src/app/files/remixd-handle.js

```diff
@@ -85,6 +90,10 @@ export class RemixdHandle extends Plugin {
           }
         }
       )
+    } else {
+      this.locahostProvider.init((error) => {
```

Comment:
Could be on one line : `this.locahostProvider.init(error => connection(error))`

Reply:
updated